core-services/ci-secret-generator/_config.yaml

```diff
@@ -195,3 +195,47 @@
     - build02
     service_account:
     - sprint-automation
+- fields:
+  - cmd: oc --kubeconfig /tmp/build-farm-credentials/sa.config-updater.$(cluster).config
+      create token -n ci $(service_account) --duration=2419200s
+    name: sa.$(service_account).$(cluster).token.txt
+  - cmd: /usr/bin/oc_create_kubeconfig.sh /tmp/build-farm-credentials $(cluster) $(service_account)
+      ci
+    name: sa.$(service_account).$(cluster).config
+  item_name: ci-operator
+  params:
+    cluster:
+    - app.ci
+    service_account:
+    - ci-operator
+- fields:
+  - cmd: /usr/bin/oc_sa_create_kubeconfig.sh /tmp/build-farm-credentials $(cluster)
```

Comment:
We will remove the reference of `/usr/bin/oc_sa_create_kubeconfig.sh`
when we work on https://issues.redhat.com/browse/DPTP-3123